src/utils/components/HidableTooltip/HidableTooltip.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC, ReactElement, ReactNode } from 'react';
+
+import { Tooltip, TooltipPosition } from '@patternfly/react-core';
+
+type HidableTooltipProps = {
+  children: ReactNode;
+  content: React.ReactNode;
+  hidden: boolean;
+  position?: TooltipPosition;
+};
+
+const HidableTooltip: FC<HidableTooltipProps> = ({
+  children,
+  content,
+  hidden,
+  position = TooltipPosition.right,
+}): ReactElement => {
```

Comment:
Why using ReactElement and not FC? Looks like FC is imported and not used, correct me if I'm wrong